intel_models/person-vehicle-bike-detection-crossroad-1016/description/person-vehicle-bike-detection-crossroad-1016.md

```diff
@@ -0,0 +1,73 @@
+# person-vehicle-bike-detection-crossroad-1016
+
+## Use case and High-level description
+
+MobileNetV2 + SSD-based network is for Person/Vehicle/Bike detection in security
+surveillance applications. Works in a variety of scenes and weather/lighting
+conditions.
+
+## Example
+
+![](./person-vehicle-bike-detection-crossroad-1016.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Mean Average Precision (mAP)    | 62.55%                                    |
+| AP people                       | 73.63%                                    |
+| AP vehicles                     | 77.84%                                    |
+| AP bikes                        | 36.18%                                    |
+| Max objects to detect           | 200                                       |
+| GFlops                          | 1.75                                      |
+| Source framework                | PyTorch                                   |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+Validation dataset consists of 34,757 images from various scenes and includes:
+
+| Type of object | Number of bounding boxes |
+|----------------|--------------------------|
+| Vehicle        | 229,503                  |
+| Pedestrian     | 240,009                  |
+| Non-vehicle    | 62,643                   |
+
+Similarly, training dataset has 219,181 images with:
+
+| Type of object | Number of bounding boxes |
+|----------------|--------------------------|
+| Vehicle        | 810,323                  |
+| Pedestrian     | 1,114,799                |
+| Non-vehicle    | 62,334                   |
+
+## Performance (FPS)
+
+### Performance disclaimer
```

Comment:
please remove `(FPS)`, `### Performance disclaimer` and `### Optimization notice`

Reply:
you didn't remove `### Performance disclaimer` and `### Optimization notice`...